app/src/androidTest/java/com/github/se/signify/ui/language/LanguageUtilsTest.kt

```diff
@@ -0,0 +1,37 @@
+package com.github.se.signify.ui.language
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import com.github.se.signify.ui.updateLanguage
+import java.util.Locale
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+class LanguageUtilsTest {
```

Comment:
Requested Change : 
Following my previous comment, you can add a test for invalid language codes to ensure that the updateLanguage function handles them correctly.

Reply:
Perfect, I should have tested edge cases thanks for pointing that